its/plugin/tests/src/integrationTest/java/com/sonar/it/php/OrchestratorTest.java

```diff
@@ -49,23 +48,23 @@
 import static java.util.Collections.singletonList;
 import static org.assertj.core.api.Assertions.assertThat;
 
-class Tests {
+class OrchestratorTest {
```

Comment:
Just having this base class seems to work. It takes the same time to run the tests before and after this change.